handlers/feedback_test.go

```diff
@@ -265,10 +241,10 @@ func Test_addFeedback(t *testing.T) {
 			addFeedback(w, req, isPositive, mockRenderer, mockSender, from, to, lang)
 
 			Convey("Then the renderer is called to render the feedback page", func() {
-				So(len(mockRenderer.DoCalls()), ShouldEqual, 1)
-				So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
-				rendererRequest := string(mockRenderer.DoCalls()[0].B)
-				So(strings.Contains(rendererRequest, `"error_type":"description"`), ShouldBeTrue)
+				So(len(mockRenderer.BuildPageCalls()), ShouldEqual, 1)
+				// So(mockRenderer.DoCalls()[0].Path, ShouldEqual, "feedback")
```

Comment:
Probably worth removing these commented out lines, line 245 to 247)

Reply:
👍